Client/src/components/Pages/Home/home.jsx

```
@@ -1,23 +1,471 @@
+          </button>
+          <div className="w-4/5 h-4/5 max-w-4xl max-h-[75vh] relative">
+            <iframe 
+              width="100%" 
+              height="100%" 
+              src="https://www.youtube.com/embed/dQw4w9WgXcQ?autoplay=1" 
+              title="Demo Video"
+              style={{ border: 0 }}
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+              allowFullScreen
+              className="z-20 shadow-2xl rounded"
+            ></iframe>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)